astropy/io/fits/tests/test_diff.py

```diff
@@ -456,6 +469,16 @@ def test_different_table_data(self):
         assert diff.diff_total == 13
         assert diff.diff_ratio == 0.65
 
+        report = diff.report()
+        assert ('Column A data differs in row 0:\n'
+                '    a> True\n'
+                '    b> False') in report
+        assert ('...and at 13 more indices.\n'
+                ' Column D data differs in row 0:') in report
+        assert ('13 different table data element(s) found (65.00% different)'
+                in report)
+        assert len(re.findall('more indices', report)) == 1
```

Comment:
haha, that explains the confusion. Both are equivalent I was just wondering why this needed regex, in the end you can use whichever you prefer.